ixnilatisApp/IxnilatisScreens.js

```diff
@@ -2,6 +2,8 @@ import React, {Component } from 'react';
 import FormWork from "./views/FormWork";
 import FormGeneralNew from "./views/FormGeneralNew";
 import FormGeneralActive from "./views/FormGeneralActive";
+import getPrivacyScreens from './views/Privacy';
```

Comment:
Please try to keep consinstent naming

`getPrivacyScreens` looks like a function name, not a component name

just `Privacy` would be better 

```suggestion
import Privacy from './views/Privacy';
```

Reply:
agree!

Reply:
note that this should change inside the file as well